tests/unittests/tests-mtd/tests-mtd.c

```diff
@@ -206,19 +206,19 @@ static void test_mtd_write_read(void)
     /* out of bounds write (addr) */
     ret = mtd_write(dev, buf, dev->pages_per_sector * dev->page_size * dev->sector_count,
                     sizeof(buf));
-    TEST_ASSERT_EQUAL_INT(-EOVERFLOW, ret);
```

Comment:
`mtd_flashpage` still uses `write` so this unittest would not work in that case would it?